.changeset/gorgeous-planes-drive.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
🙃 why not adding a changelog?

Reply:
It's irrelevant for the package and also doesn't hurt to stay until the next release. No reason to clutter the changelog with it as it's not important for users.